server/lib/buff_server/packages.ex

```diff
@@ -50,7 +53,10 @@ defmodule BuffServer.Packages do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_package(attrs \\ %{}) do
+  def create_package(attrs \\ %{}, artifact_binary) do
+    bucket_path = "/#{attrs.name}/artifact"
```

Comment:
attrs.name at this point isnt validated to be a valid package name..
I would run it through the changeset first(without actually inserting to the db yet). make sure its valid. and only then create the bucket.

Reply:
hmm what am I missing? its run thru the changeset